config/initializers/dfe_analytics.rb

```diff
@@ -54,13 +54,15 @@
   # use a new version of the BigQuery streaming APIs.
   config.azure_federated_auth = true
 
+  # Path of airbyte stream config file relative to the App root (Rails.root)
+  config.airbyte_stream_config_path = "terraform/aks/workspace-variables/airbyte_stream_config.json"
+
+  config.airbyte_enabled = true if Rails.env.development?
+
   if Rails.env.in?(%w[review qa]) && ENV["BIGQUERY_AIRBYTE_DATASET"].present?
     # Perform airbyte checks on startup and allow airbyte config generation
     config.airbyte_enabled = true
 
-    # Path of airbyte stream config file relative to the App root (Rails.root)
-    config.airbyte_stream_config_path = "terraform/aks/workspace-variables/airbyte_stream_config.json"
-
     # Set bigquery airbyte vars
     config.bigquery_hidden_policy_tag = "projects/rugged-abacus-218110/locations/europe-west2/taxonomies/69524444121704657/policyTags/6523652585511281766"
   end
```

Comment:
I know this is somewhat temporary as we gradually roll out airbyte to all envs, but could this be refactored a bit for clarity and de-duping?

```suggestion
  if Rails.env.in?(%w[development review qa])
    # Path of airbyte stream config file relative to the App root (Rails.root)
    config.airbyte_stream_config_path = "terraform/aks/workspace-variables/airbyte_stream_config.json"

    # Perform airbyte checks on startup and allow airbyte config generation
    config.airbyte_enabled = true

    # Set bigquery airbyte vars
    config.bigquery_hidden_policy_tag = "projects/rugged-abacus-218110/locations/europe-west2/taxonomies/69524444121704657/policyTags/6523652585511281766" if ENV["BIGQUERY_AIRBYTE_DATASET"].present?
  end
end
```